exercises/square-root/introduction.md

```diff
@@ -0,0 +1,10 @@
+# Introduction
+
+We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.
+
+As the first step in our calculation, we take a number and find its square root—that is, the number that, when multiplied by itself, equals the given number.
+
+The journey will be very long.
+To make the batteries last as long as possible, we had to make our rocket’s onboard computer very power efficient.
+Unfortunately that means that we can’t rely on fancy math libraries and functions, as they use more power.
+Instead we want to calculate or approximate the square root directly.
```

Comment:
Are we calculating or approximating? If we're approximating I think we need that to be clearer in the statement above.